internal/protocol/request.go

```diff
@@ -45,27 +45,47 @@ func EncodePrepare(request *Message, db uint64, sql string) {
 	request.putUint64(db)
 	request.putString(sql)
 
-	request.putHeader(RequestPrepare)
+	request.putHeader(RequestPrepare, 0)
 }
 
-// EncodeExec encodes a Exec request.
-func EncodeExec(request *Message, db uint32, stmt uint32, values NamedValues) {
+// EncodeExecV0 encodes a Exec request.
```

Comment:
This comment doesn't highlight the difference between `EncodeExecV0` and `EncodeExecV1`---the choice between each is unclear. As this comes up in quite a few places, is it worth putting a doc at the top of the file to establish the convention? Perhaps something like:
```
// Many functions in this file have two versions, distinguished
// by their suffix. Those named FooV0 handle queries which accept
// at most 255 arguments. Those named FooV1 handle queries with
// arbitrarily-many arguments but ...